### ISSUE TITLE
Tooltip.js demo

### ISSUE THREAD

**Issue by maxkfranz** [cytoscape/cytoscape.js-popper#12]
It would be nice to have a small demo using the Popper Tooltip.js lib: https://popper.js.org/tooltip-documentation.html

Maybe just one node that shows a tooltip on click like this one: http://js.cytoscape.org/demos/qtip-extension/

It would also be nice to have a few lines of example code for Tooltip.js in the readme.

**Comment by hmtinc** [comment]
@maxkfranz 

I looked into the documentation and the initialization function `new Tooltip(reference, options)` requires `reference` to be an HTML Element. This requirement is enforced because the tooltip library internally attempts to add event listeners to the passed references, and the reference object from this extension does not contain the ability to be used in place of a HTML element.  

This is identical to the issue we encountered with tippy.js, except tippy.js was modified to support plain reference objects.

I was able to get tooltip.js to work by adding the following to `.popperRef`
```js
     //Dummy event listener and attribute functions
     //This allows the reference object to be used in place of an HTML element
        attributes : {},
        parentNode : document.body,
        addEventListener : () => true,
        removeEventListener : () => true,
        setAttribute : (key, val) => refObject.attributes[key] = val,
        getAttribute : (key) => refObject.attributes[key]
```
This doesn't seem like an ideal solution as the referenceObject should only ever provide 3 fields (`getBoundingRect, getWidth, getHeight`) and adding more violates the standard defined in the main popper.js docs.
